webapp/src/components/Poll.vue

```diff
@@ -6,6 +6,7 @@
 	template(v-else)
 		.option(v-for="option of poll.options", :class="{'most-votes': optionsWithMostVotes.includes(option.id)}")
 			.content {{ option.content }}
```

Comment:
```suggestion
			.content
				| {{ option.content }}
```

Would be clearer that there are two children in `.content`